mne/io/eyelink/_utils.py

```diff
@@ -0,0 +1,107 @@
+"""Helper functions for reading eyelink ASCII files."""
+# Authors: Scott Huberty <seh33@uw.edu>
+# License: BSD-3-Clause
+
+import re
+
+from ...preprocessing.eyetracking.calibration import Calibration
+
+
+def _find_recording_start(lines):
+    """Return the first START line in an SR Research EyeLink ASCII file.
+
+    Parameters
+    ----------
+        lines: A list of strings, which are The lines in an eyelink ASCII file.
+
+    Returns
+    -------
+        The line that contains the info on the start of the recording.
+    """
+    for line in lines:
+        if line.startswith("START"):
+            return line
+    raise ValueError("Could not find the start of the recording.")
+
+
+def _parse_validation_line(line):
+    """Parse a single line of eyelink validation data.
+
+    Parameters
+    ----------
+        line: A string containing a line of validation data from an eyelink
+        ASCII file.
+
+    Returns
+    -------
+        A list of tuples containing the validation data.
+    """
+    tokens = line.split()
+    xy = tokens[-6].strip("[]").split(",")  # e.g. '960, 540'
+    xy_diff = tokens[-2].strip("[]").split(",")  # e.g. '-1.5, -2.8'
+    vals = tuple([float(v) for v in [*xy, tokens[-4], *xy_diff]])
+
+    return vals
+
+
+def _parse_calibration(
+    lines, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Parse the lines in the given list and returns a list of Calibration instances.
+
+    Parameters
+    ----------
+        lines: A list of strings, which are The lines in an eyelink ASCII file.
+
+    Returns
+    -------
+        A list containing one or more Calibration instances,
+        one for each calibration that was recorded in the eyelink ASCII file
+        data.
+    """
```

Comment:
Like this, and remove it from the top
```suggestion
    """
    from ...preprocessing.eyetracking.calibration import Calibration
```